src/components/CivicLikerV3/PureDashboard.vue

```diff
@@ -169,22 +169,32 @@
       />
     </section>
 
-    <section class="mt-32">
+    <section class="mt-[64px]">
       <i18n
         class="text-24 text-like-green"
         path="civic_dashboard_v3_intro_title"
         tag="h1"
       />
       <div class="p-32 mt-32 overflow-hidden bg-white rounded-8">
         <img
-          src="~/assets/images/civic-v3/banner.png"
+          class="rounded-[8px]"
+          src="/images/og/writing-nft.jpg"
           :alt="$t('civic_dashboard_v3_intro_title')"
         >
         <i18n
-          class="mt-12 px"
+          class="mt-32"
           path="civic_dashboard_v3_intro_description"
           tag="p"
         />
+
+        <div class="flex justify-center mt-[32px]">
+          <ButtonV2
+            :text="$t('about_nft_page_title')"
+            preset="secondary"
+            to="/campaign/writing-nft"
```

Comment:
Should we prefer named route?

Reply:
I prefer simply path when no params/ query

Reply:
hard to find and replace